DemoViper/Sources/Modules/ListRepos/View/ReposViewController.swift

```diff
@@ -61,12 +61,12 @@ extension ReposViewController: ReposViewInput {
 
 // MARK: - Setup UI
 extension ReposViewController {
-    private func setupUI() {
-        setupLayouts()
-        setupViews()
+    private func setUpUI() {
```

Comment:
This function looks verbose. Considering removing this and move `setUpLayout` and `setUpViews` to `configure()`. BTW `setUpLayout` not `setUpLayouts` cuz we have only one layout, right.